Docker/python-nginx/python2.7-alpine3.7/dockerrun.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/sh
+#
+# Note: base alpine Linux image may not include bash shell,
+#    and we probably want to move to that for service images,
+#    so just use bourn shell ...
+
+#
+# Update certificate authority index -
+# environment may have mounted more authorities
+# - ex: /usr/local/share/ca-certificates/cdis-ca.crt into system bundle       
+#
+
+GEN3_DEBUG="${GEN3_DEBUG:-False}"
+GEN3_SIDECAR="${GEN3_SIDECAR:-False}"
+GEN3_DRYRUN="${GEN3_DRYRUN:-False}"
```

Comment:
Add a test step to the Jenkinsfile to run `dockerrun.sh` with `GEN3_DRYRUN` enabled ...